### ISSUE TITLE
Using reef.polyfills.es causes "Proxy polyfill does not support trap 'deleteProperty'" error in IE 11

### ISSUE THREAD

**Issue by brudolph** [cferdinandi/reef#59]
There is no codepen since it can't be tested in IE.

I was getting an error in IE when using the pollyfill version of Reef.

`Unhandled promise rejection TypeError: Proxy polyfill does not support trap  'deleteProperty' `

After some digging online it looks like most Proxy polyfills do not support deleting object properties. Only get and set. So for my situation I commented out line 166 through 170 

```javascript
deleteProperty: function (obj, prop) {
    delete obj[prop];
    debounceRender(instance);
    return true;
}
```

But then I was getting this error: 

`Unable to get property 'childNodes' of undefined or null reference`

After debugging in IE, the error was on line 647, inside of the `stringToHTML` function. For reasons I couldn't figure out, the `head` element was returning a null value. For my situation I commented out the following if statement inside of the `stringToHTML` function.

```javascript
if (support) {

  // Create document
  var parser = new DOMParser();
  var doc = parser.parseFromString(str, 'text/html');

  // If there are items in the head, move them to the body
  if (doc.head.childNodes.length > 0) {
    Array.prototype.slice.call(doc.head.childNodes).reverse().forEach(function (node) {
      doc.body.insertBefore(node, doc.body.firstChild);
    });
  }

  return doc.body;

}
```

Forcing Reef to use the old school method.

```javascript
var dom = document.createElement('div');
dom.innerHTML = str;
return dom;
```

**Comment by cferdinandi** [comment]
It's like you read my mind. I was just looking into this today!

**Comment by brudolph** [comment]
Let me know if you need me to test anything on PC. I'd be happy to help out!

**Comment by cferdinandi** [comment]
Fortunately, I have access to a windows machine for testing, but thanks!

This *should* be fixed with v7.3.2 (including the `stringToHTML()` issue). Let me know if it's still not working for you, though.
